.readthedocs.yaml

```diff
@@ -0,0 +1,20 @@
+version: 2
```

Comment:
According to RTD, it can't:
> The [configuration file](https://docs.readthedocs.io/en/stable/config-file/index.html) must be in the root directory of your project and be named .readthedocs.yaml.

https://docs.readthedocs.io/en/stable/config-file/v2.html